src/main.ml

```diff
@@ -147,12 +147,25 @@ let make_message_switch_server () =
   | `Error (`Msg m) ->
     Lwt.fail_with (Printf.sprintf "Failed to listen on message-switch queue: %s" m)
 
+let get_log_level () =
```

Comment:
We usually define a command here, so perhaps call this `let cmd = `, and move it below such that `main` is already in scope.